specs/src/specs/shares.md

```diff
@@ -0,0 +1,90 @@
+# Shares
+
+<!-- toc -->
+
+## Overview
+
+All available data in a Celestia block is split into fixed-size data chunks known as "shares". A share is associated with exactly one namespace. The shares in a Celestia block are eventually erasure-coded and committed to in [Namespace Merkle trees](./data_structures.md#namespace-merkle-tree).
+
+A share sequence is a contiguous set of shares that contain semantically relevant data. A share sequence may contain one or more shares. In most cases, a share sequence should be parsed together because the original data may have been split across share boundaries. One share sequence exists per [reserved namespace](./consensus.md#reserved-namespaces) and per [blob](../../../x/blob/README.md).
+
+## Share Structure
+
+For shares **with a namespace above [`MAX_RESERVED_NAMESPACE`](./consensus.md#constants) but below [`PARITY_SHARE_NAMESPACE`](./consensus.md#constants)**:
+
+- The first [`NAMESPACE_SIZE`](./consensus.md#constants) of a share's raw data is the namespace of that share.
+- The next [`SHARE_INFO_BYTES`](./consensus.md#constants) bytes are for share information with the following structure:
+  - The first 7 bits represent the share version in big endian form (initially, this will be `0000000` for version `0`);
+  - The last bit is a sequence start indicator. The indicator is `1` if the share is the first share of a sequence or `0` if the share is a continuation share of a sequence.
+- If this is the first share of a sequence the next [`SEQUENCE_BYTES`](./consensus.md#constants) contain a big endian `uint32` that represents the length of the sequence that follows in bytes.
+- The remaining [`SHARE_SIZE`](./consensus.md#constants)`-`[`NAMESPACE_SIZE`](./consensus.md#constants)`-`[`SHARE_INFO_BYTES`](./consensus.md#constants)`-`[`SEQUENCE_BYTES`](./consensus.md#constants) bytes (if first share) or [`SHARE_SIZE`](./consensus.md#constants)`-`[`NAMESPACE_SIZE`](./consensus.md#constants)`-`[`SHARE_INFO_BYTES`](./consensus.md#constants) bytes (if continuation share) are blob data. Blob data are opaque bytes of data that are included in the block but do not impact the state. In other words, the remaining bytes have no special meaning and are simply used to store data.
```

Comment:
> Blob data are opaque bytes of data that are included in the block but do not impact the state. In other words, the remaining bytes have no special meaning and are simply used to store data.

I think if we add the "Overview" section as per my previous comment (in which we would explain the concept blob Tx  and that it carries some blobs), then we can remove these lines from here (they are not directly related to the share encoding imo, for example the term "state" is not mentioned in earlier parts and may cause confusion as to why we care about the state in the share encoding stage).

Reply:
In an effort to minimize the scope of this page, I think blob Tx should be explained on the blob page rather than the specs page. Will remove the term "state"

Reply:
Resolved via 4fd7ebee1d4f6016c507fe8f0f5d83477ecba92f and 6e99898b0ed5c9a669e5e133e049403064636736

Reply:
>  I think blob Tx should be explained on the blob page rather than the shares page.

Fine with that, yet, to enhance the clarity of the document, it would be beneficial to briefly mention that "blobs" refer to data that BlobTxs initially carried but were subsequently separated from the original transaction. We can then give a link to the respective specs.